app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala

```diff
@@ -86,6 +86,42 @@ object Projects extends TableQuery(tag => new Projects(tag)) with LazyLogging {
     }
   }
 
+  /** Get the AOIs belonging to a project. */
+  def listAOIs(
+    projectId: UUID,
+    pageRequest: PageRequest,
+    user: User
+  )(implicit database: DB): Future[PaginatedResponse[AOI]] = {
+    // TODO: This should be a join?
+    val aois: Future[Seq[UUID]] = database.db.run {
+      AoisToProjects
+//        .filterUserVisibility(user)  // TODO!
```

Comment:
For this one, maybe not any for now since it already is filtered by project. If we think of more we can add them later.

I mentioned it above, but adding `filterToSharedOrganizationIfNotInRoot` is how we will have to filter for permissions/authorization because `VISIBILITY` isn't on AOIs